apps/web/app/(public)/(projects)/projects/project-card.tsx

```
@@ -23,13 +38,18 @@ export default function ProjectCard({ project }: { project: Project }) {
       >
         <span className="sr-only">View {project.name}</span>
         <div className="mb-4 flex items-start gap-3">
-          <Image
-            src={repo?.owner.avatar_url ?? 'https://placehold.co/48x48'}
-            alt={project.name ?? 'Project Logo'}
-            width={48}
-            height={48}
-            className="h-20 w-20 rounded-full"
-          />
+          {(repo && repo?.owner && repo?.owner?.avatar_url) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)